src/main/java/duke/task/DeadlineTask.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class handles the deadlineTasks
+ */
+public class DeadlineTask extends Task {
+    private LocalDate date;
+
+    public DeadlineTask(String task) {
+        super(task);
+    }
+
+    String[] divideCommand = task.split(" ");
```

Comment:
Perhaps a clearer name would be better, also maybe this string array could be placed in toString() since it is not needed anywhere else?